<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="3" total="4" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-04-21T21:27:43 CEST" name="Suite" finished-at="2024-04-21T21:27:47 CEST" duration-ms="4538">
    <groups>
    </groups>
    <test started-at="2024-04-21T21:27:43 CEST" name="JavascriptExercise" finished-at="2024-04-21T21:27:47 CEST" duration-ms="4538">
      <class name="AdvancedExercises">
        <test-method is-config="true" signature="setup()[pri:0, instance:AdvancedExercises@4310d43]" started-at="2024-04-21T21:27:43 CEST" name="setup" finished-at="2024-04-21T21:27:45 CEST" duration-ms="2386" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="startTest()[pri:0, instance:AdvancedExercises@4310d43]" started-at="2024-04-21T21:27:45 CEST" name="startTest" finished-at="2024-04-21T21:27:45 CEST" duration-ms="295" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startTest -->
        <test-method is-config="true" signature="initializePage()[pri:0, instance:AdvancedExercises@4310d43]" started-at="2024-04-21T21:27:45 CEST" name="initializePage" finished-at="2024-04-21T21:27:45 CEST" duration-ms="144" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializePage -->
        <test-method signature="TestUsingImageRecognition()[pri:0, instance:AdvancedExercises@4310d43]" started-at="2024-04-21T21:27:45 CEST" name="TestUsingImageRecognition" finished-at="2024-04-21T21:27:47 CEST" duration-ms="1037" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at AdvancedExercises.TestUsingImageRecognition(AdvancedExercises.java:147)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestUsingImageRecognition -->
        <test-method is-config="true" signature="endTest()[pri:0, instance:AdvancedExercises@4310d43]" started-at="2024-04-21T21:27:47 CEST" name="endTest" finished-at="2024-04-21T21:27:47 CEST" duration-ms="477" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
        <test-method is-config="true" signature="quit()[pri:0, instance:AdvancedExercises@4310d43]" started-at="2024-04-21T21:27:47 CEST" name="quit" finished-at="2024-04-21T21:27:47 CEST" duration-ms="152" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- AdvancedExercises -->
    </test> <!-- JavascriptExercise -->
  </suite> <!-- Suite -->
</testng-results>
